pkg/scheduler/core/generic_scheduler.go

```diff
@@ -492,7 +535,7 @@ func (g *genericScheduler) prioritizeNodes(
 		}
 	}
 
-	if klog.V(10).Enabled() {
+	if klog.V(4).Enabled() {
```

Comment:
why change this?

Reply:
I think this information matches well with the info that I'm exposing in this PR. The highest scoring plugins themselves are a direct contributor to the total score for that node, so I'm lowering it from `V(10)`, which is the level at which we log every score for every plugin.

Reply:
sgtm